main/src/ca/uwaterloo/flix/language/ast/shared/Annotation.scala

```diff
@@ -150,4 +150,27 @@ object Annotation {
     override def toString: String = "@" + name
   }
 
+  /**
+    * An AST node that represents an `@Inline` annotation.
+    *
+    * A function marked with `@Inline` is guaranteed to be inlined by the compiler.
+    * If it is recursive, it is only ever unrolled once into the same function.
```

Comment:
-> If the function is recursive, it may be unrolled in undetermined number of times.

Reply:
(Right-- because each round would unroll it once?)